components/CardList.tsx

```diff
@@ -0,0 +1,28 @@
+import Card from "@/components/Card";
+import styles from "@/styles/CardList.module.css";
+
+interface CardListInterface {
+  id: number;
+  created_at: string;
+  updated_at: null;
+  url: string;
+  title: string;
+  description: string;
+  image_source: string;
+  folder_id: number;
+}
+
+type CardListProps = {
+  selectedFolderData: CardListInterface[];
```

Comment:
-Data와 같은 접미사는 필요없을 것 같네요!
```suggestion
  selectedFolder: CardListInterface[];
```